test/mlir/onnx/onnx_decompose_einsum.mlir

```diff
@@ -107,12 +107,12 @@ func.func @test_einsum_ibh_hnd(%arg0: tensor<128x1x1024xf16>, %arg1: tensor<1024
   return %0 : tensor<128x1x16x64xf16>
 // CHECK-LABEL:  func.func @test_einsum_ibh_hnd
 // CHECK-SAME:   ([[PARAM_0_:%.+]]: tensor<128x1x1024xf16>, [[PARAM_1_:%.+]]: tensor<1024x16x64xf16>) -> tensor<128x1x16x64xf16> {
-// CHECK:           [[VAR_0_:%.+]] = "onnx.Constant"() {value = dense<[128, 1024]> : tensor<2xi64>} : () -> tensor<2xi64>
+// CHECK:           [[VAR_0_:%.+]] = onnx.Constant dense<[128, 1024]> : tensor<2xi64>
 // CHECK-DAG:       [[VAR_1_:%.+]] = "onnx.Reshape"([[PARAM_0_]], [[VAR_0_]]) {allowzero = 0 : si64} : (tensor<128x1x1024xf16>, tensor<2xi64>) -> tensor<128x1024xf16>
-// CHECK-DAG:       [[VAR_2_:%.+]] = "onnx.Constant"() {value = dense<1024> : tensor<2xi64>} : () -> tensor<2xi64>
+// CHECK-DAG:       [[VAR_2_:%.+]] = onnx.Constant dense<1024> : tensor<2xi64>
```

Comment:
@tungld here is an example of a splat constant

Reply:
I see. Thanks!